### PR TITLE
Set up config for frontend

### PR DESCRIPTION
**Pull request by GodVenn**

### CODE REVIEW

**Comment by GodVenn** [comment]
Note that the PR seems to change more than it does because i properly configured prettier to run "on save" on my laptop while working on this

File: frontend/.env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+PORT=3001
+REACT_APP_BACKEND_URL=https://localhost:8000
+REACT_APP_FRONTEND_URL=http://localhost:3001
+REACT_APP_AD_CLIENT_ID=f5993820-b7e2-4791-886f-f9f5027dc7be
+REACT_APP_AD_TENANT_ID=3aa4a235-b6e2-48d5-9195-7fcf05b459b0
```
Comment (oysand) [inline frontend/.env.example:5]: Missing new line at the end

File: frontend/src/config.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    else
+        return value as string
+}
+
+export const config = {
+    BACKEND_URL: GetEnvVariable("REACT_APP_BACKEND_URL"),
+    FRONTEND_URL: GetEnvVariable("REACT_APP_FRONTEND_URL"),
+    AD_CLIENT_ID: GetEnvVariable("REACT_APP_AD_CLIENT_ID"),
+    AD_TENANT_ID: GetEnvVariable("REACT_APP_AD_TENANT_ID"),
+}
```
Comment (oysand) [inline frontend/src/config.ts:17]: Missing new line

File: frontend/package.json [hunk 3]
```diff
@@ -27,7 +27,7 @@
         "prettier": "^2.4.1"
     },
     "scripts": {
-        "start": "PORT=3001 react-scripts start",
+        "start": "react-scripts start",
```
Comment (oysand) [inline frontend/package.json:30]: The docker container is set up to run with port 3001

**Comment by oysand** [comment]
Yes, think we can merge it

**Pull request merged**